### DISCUSSION TITLE
bulk insert support of asyncmy

### DISCUSSION THREAD

**Discussion by whitemay** [encode/databases#509] [Ideas]
I noticed that asyncmy now support executemany to improve performance. 
I think it is time to use executemany now. bulk inerting sooooooooo slow.

**Comment by aminalaee** [comment]
The execute_many right now is running execute in a loop: https://github.com/encode/databases/blob/master/databases/backends/postgres.py#L226

Feel free to create a PR for that driver or any other that supports it.
